api/app/urls.py

```diff
@@ -16,14 +18,21 @@
         r"^sales-dashboard/",
         include("sales_dashboard.urls", namespace="sales_dashboard"),
     ),
-    url(r"", lambda r: HttpResponse("Flagsmith API")),
     # this url is used to generate email content for the password reset workflow
     url(
         r"^password-reset/confirm/(?P<uidb64>[0-9A-Za-z_\-]+)/(?P<token>[0-9A-Za-z]{1,"
         r"13}-[0-9A-Za-z]{1,20})/$",
         password_reset_redirect,
         name="password_reset_confirm",
     ),
+    url(
+        r"^api/project-overrides",
```

Comment:
Why is this an `api` route?

Reply:
yeah I agree on this one - @kyle-ssg we should agree on a better URL namespace that doesn't include `api` - maybe `settings/project-overrides`?

Reply:
I don't really mind. Just let me know the URL before you merge this so I can update saas.

Reply:
OK I've updated that now to use `/config/project-overrides` on both django and node @kyle-ssg